addons/pos_event/static/src/app/models/pos_order_line.js

```diff
@@ -18,7 +18,7 @@ patch(PosOrderline.prototype, {
             };
         } else if (this.event_ticket_id) {
             for (const registration of this.event_registration_ids) {
-                registration.delete();
+                registration.delete({ silent: true });
```

Comment:
From where saved registration are deleted, if we don't do the server call here ?

Reply:
@davidmonnom When the orderline is removed, the registration will also be deleted.

Reply:
pefect :-)